File: Iniciante/URI 1020.pas
program uri_1020;
uses crt;
const
	ano = 365;
	mes = 30;
	lnb = #13#10;
var
	anos,meses,dias : real;

BEGIN
	readln(dias);
	anos:= trunc(dias/ano);
	dias:= dias - (anos*ano);
	meses:= trunc(dias/mes);
	dias:= dias - (meses*mes);
	writeln(anos:0:0,' ano(s)',lnb,
			meses:0:0,' mes(es)',lnb,
			dias:0:0,' dia(s)');
END.

File: Iniciante/URI 1020.c
#include <stdio.h>

int main () {

  unsigned short int x, ano, mes;
  scanf("%hd", &x);

  ano = x/365;
  x -= (ano*365);

  mes = x/30;
  x -= (mes*30);

  printf("%d ano(s)\n", ano);
  printf("%d mes(es)\n", mes);
  printf("%d dia(s)\n", x);

}